breadcord/bot.py

```diff
@@ -49,6 +50,7 @@ async def close(self) -> None:
     def discover_modules(self) -> None:
         modules = []
         for search_location in Path('breadcord/core_modules'), Path('breadcord/modules'):
+            Path(search_location).mkdir(exist_ok=True)
```

Comment:
`search_location` is already a `pathlib.Path` object, there's no need to wrap it again.